src/components/home/header.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import logo from '../../images/logo_white.png';
+import userImage from '../../images/default-user-image.jpg';
+
+const Header = () => {
+  return (
+    <div className="home-header">
```

Comment:
@noor-ahmed Should we use `app-header` instead of `home-header` as we will use this component in another screen/page in nearly future? 😛